REQUIRE

```diff
@@ -1,4 +1,4 @@
-julia 0.7.0-beta.85
+julia 1.0
```

Comment:
unless you're using deprecated functionality, probably better to make the `julia 0.7` to make it easier on people upgrading packages.

Reply:
doesn't it set an upper bound too?

Reply:
no, you use 2 numbers to specify an upper bound.